src/DonorProfiles/RequestHandler.php

```diff
@@ -2,8 +2,9 @@
 
 namespace Give\DonorProfiles;
 
-use Give\DonorProfiles\App;
+use _WP_Dependency;
```

Comment:
What is the reasoning/need for this change?

Reply:
I import this class because we are using this class in code.
![image](https://user-images.githubusercontent.com/1784821/110483866-22e7ab80-8110-11eb-9717-ca531ec1d711.png)